frontend/src/routes/static/aktuelles/+page.svelte

```diff
@@ -1,3 +1,9 @@
-<div class="p-10">
```

Comment:
It might make sense to move this out of the static folder now that it's no longer just a static html page?